pkgs/os-specific/linux/sgx-sdk/default.nix

```diff
@@ -46,25 +46,26 @@ stdenv.mkDerivation rec {
       ./linux/installer/common/sdk/install.sh
   '';
 
-  dontConfigure = true;
+  # We need `cmake` as a build input but don't use it to kick off the build phase
+  dontUseCmakeConfigure = true;
 
   # SDK built with stackprotector produces broken enclaves which crash at runtime.
   # Disable all to be safe, SDK build configures compiler mitigations manually.
   hardeningDisable = [ "all" ];
 
   nativeBuildInputs = [
+    autoconf
+    automake
     cmake
+    file
     git
+    nasm
+    ncurses
```

Comment:
How is ncurses used in the build?

Reply:
Very good question. I just built the derivation without it and it turns out: it is not required. Going to remove it.